analyzers/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -3,13 +3,13 @@
   <PropertyGroup>
     <AssemblyName>SonarAnalyzer.CSharp</AssemblyName>
     <RootNamespace>SonarAnalyzer</RootNamespace>
-    <!-- See SonarAnalyzer.Common.csproj for more details-->
-    <TargetFrameworks Condition="$(BuildOnlyNet46) != 'true'">net46;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="$(BuildOnlyNet46) == 'true'">net46</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- .NET Standard target does not copy referenced DLLs into bin folder, so we need to enable it explicitly. -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
Can't we put this into the `Directory.Build.targets`?

Reply:
No, we need it only for 2 projects out of 7